doc/api/next_api_changes/deprecations/28842-ES.rst

```diff
@@ -0,0 +1,53 @@
+ft2font module-level constants replaced by enums
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The `.ft2font`-level constants have been converted to `enum` classes, and all API using
+them now take/return the new types.
+
+The following constants are now part of `.ft2font.Kerning`:
+
+- ``KERNING_DEFAULT``
```

Comment:
perhaps note that the new name doesn't include KERNING_; ditto for LOAD_.

Reply:
Done.